x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ldap/PoolingSessionFactory.java

```diff
@@ -201,8 +200,8 @@ static LDAPConnectionPool createConnectionPool(
                     pool.setHealthCheckIntervalMillis(healthCheckInterval);
                 } else {
                     logger.warn(
-                        new ParameterizedMessage(
-                            "[{}] and [{}} have not been specified or are not valid distinguished names,"
+                        () -> format(
+                            "[%s] and [%s} have not been specified or are not valid distinguished names,"
```

Comment:
I think the closing `}` in the second parameter is a typo. We should probably fix it.